content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,46 +2,168 @@
+   selection is made, this policy will apply to all ticket types.
+ - :guilabel:`Tags`: Tags are applied to briefly indicate what the ticket is about. Multiple tags
+   can be applied to a single ticket.
+ - :guilabel:`Customers`: Individual contacts or companies may be selected in this field.
+ - :guilabel:`Sales Order Items`: This field is available only if a team has :guilabel:`Timesheets`
+   enabled. This allows the ticket to tie directly to a specific line on a sales order, which must
+   be indicated on the ticket in the :guilabel:`Sales Order Item` field.
+
+ .. note::
+    Unless otherwise indicated, multiple selections can be made for each field. (i.e.- multiple
+    :guilabel:`Tags` can be included in a policy, but only one :guilabel:`Priority` level)
+
+.. image:: sla/sla-create-new.png
+   :align: center
+   :alt: View of a blank SLA policy record
+
+Establishing a target for an SLA policy
+---------------------------------------
+
+The :guilabel:`Target` is the stage a ticket needs to reach, and the time alloted to reach that
+stage, in order to satisfy the SLA policy. Any stage assigned to a team may be selected for the
+:guilabel:`Reach Stage` field. Time spent in stages selected in :guilabel:`Excluding Stages` will
+not be included in the calculation of the SLA deadline.
+
+.. example::
+   An SLA titled `8 Hours to Close` tracks the working time before a ticket is completed, and would
+   have `Solved` as the :guilabel:`Reach Stage`. However, if the SLA was titled `2 Days to Start`,
+   it tracks the working time before work on a ticket has begun, and would have `In Progress`
+   as the :guilabel:`Reach Stage`.
+
+Meeting SLA deadlines
+=====================
+
+Once it is determined that a ticket fits the criteria of an SLA policy, a deadline is calculated.
+The deadline is based on the creation date of the ticket, and the targeted working hours. The
+deadline is then added to the ticket, as well as a white tag indicating the name of the SLA applied.
+
+.. image:: sla/sla-open-deadline.png
+   :align: center
+   :alt: View of a ticket's form emphasizing an open SLA deadline on a ticket in Odoo Helpdesk
+
+.. important::
+   If a ticket fits the criteria for more than one SLA, the earliest occurring deadline will be
+   displayed on the ticket. Once that deadline has passed, the next deadline will be displayed.
+
+Once a ticket satisfies an SLA policy, the SLA tag turns green, and the deadline field disappears.
+
+.. image:: sla/sla-deadline.png
    :align: center
-   :height: 330
-   :alt: View of a ticket’s form emphasizing a satisfied SLA in Odoo Helpdesk
+   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk
+
+If the SLA deadline passes and the ticket has not moved to the :guilabel:`Reach Stage`, the SLA tag
+will turn red. Once the SLA has failed, the red tag will stay on the ticket, even after the ticket
+is moved to the reach stage.
```

Comment:
:guilabel:`Reach Stage` - stay consistent with the prior use of :guilabel:`Reach Stage`